DataFormats/Portable/interface/PortableHostCollection.h

```diff
@@ -94,7 +94,7 @@ class PortableHostCollection {
     layout.ROOTStreamerCleaner();
   }
 
-  void aggregate(ConstView const& view) { layout_.aggregate(view); }
+  void deep_copy(ConstView const& view) { layout_.deep_copy(view); }
```

Comment:
All other methods use camelCase, so better use `deepCopy`.

Reply:
@makortel can I ask your opinion here ?

One option is to merge this PR with the current implementation of `deepCopy(ConstView const& view)` (possibly renamed to `deepCopyFrom(...)`).
The problem is that we cannot guarantee that the `View` points to host memory.

An other option is to implement the alpaka-based copies, but that seems to require some kind of extension to the `View` types to know the (alpaka) device that holds the memory.
In that case, too, it might be worthwhile to have a faster host-only copy that doesn't use alpaka, queues, _etc_.

Do you think we should go ahead with the simple, host-only deep copy - or wait an implement the full-fledged alpaka based one ?

Reply:
OK, thanks.
@leobeltra can you add a comment before the `deepCopy` method, along the lines of
```c++
  /* Copy column by column the content of the given view into this PortableHostCollection.
   * The view must point to data in host memory.
   */
```